File: Modules/Common/Resources/SIE.Web.Resources/Employees/Commands/EmployeeSaveCommand.js
SIE.defineCommand('SIE.Web.Resources.Employees.Commands.EmployeeSaveCommand', {
    extend: 'SIE.cmd.FormSave',
    meta: { text: "保存", group: "edit", iconCls: "icon-SaveEntity icon-blue" },
    onSaved: function (view, res) {
        this.callParent(arguments);
        var me = this;
        var control = me.view.getControl().query('[name=Code]')[0];
        control["setReadOnly"](true);
    }
});

File: Modules/Common/Resources/SIE.Web.Resources/Employees/Commands/EmployeeSaveCommand.cs
using SIE.Domain;
using SIE.Resources.Employees;
using SIE.Web.Command;

namespace SIE.Web.Resources.Employees.Commands
{
    /// <summary>
    /// 员工保存命令
    /// </summary>
    [JsCommand("SIE.Web.Resources.Employees.Commands.EmployeeSaveCommand")]
    public class EmployeeSaveCommand : FormSaveCommand
    {
        /// <summary>
        /// 保存员工
        /// </summary>
        /// <param name="entity"></param>
        protected override void DoSave(Entity entity)
        {
            Employee data = entity as Employee;
            RT.Service.Resolve<EmployeeController>().SaveEditedEmployee(data);
        }
    }
}
